sdk/spring/spring-reference.yml

```diff
@@ -508,29 +508,29 @@
               - compatibilityRange: "[3.0.0,3.0.13]"
                 groupId: com.azure.spring
                 artifactId: spring-cloud-azure-starter-keyvault-certificates
-                version: 5.19.0
+                version: 5.20.0
               - compatibilityRange: "[3.1.0,3.1.12]"
                 groupId: com.azure.spring
                 artifactId: spring-cloud-azure-starter-keyvault-certificates
-                version: 5.19.0
+                version: 5.20.0
               - compatibilityRange: "[3.2.0,3.2.12]"
                 groupId: com.azure.spring
                 artifactId: spring-cloud-azure-starter-keyvault-certificates
-                version: 5.19.0
-              - compatibilityRange: "[3.3.0,3.3.6]"
+                version: 5.20.0
+              - compatibilityRange: "[3.3.0,3.3.8]"
                 groupId: com.azure.spring
                 artifactId: spring-cloud-azure-starter-keyvault-certificates
-                version: 5.19.0
-              - compatibilityRange: "[3.4.0,3.4.0]"
+                version: 5.20.0
+              - compatibilityRange: "[3.4.0,3.4.2]"
                 groupId: com.azure.spring
                 artifactId: spring-cloud-azure-starter-keyvault-certificates
-                version: 5.19.0
+                version: 5.20.0
         - artifactId: azure-security-keyvault-jca
           groupId: com.azure
           versionGA: 2.10.0
           type: client
           links:
-            javadoc: https://azuresdkdocs.z19.web.core.windows.net/java/azure-security-keyvault-jca/latest/index.html
+            javadoc: https://azuresdkdocs.blob.core.windows.net/$web/java/azure-security-keyvault-jca/2.10.0/index.html
```

Comment:
These links can no longer be accesse via `https://azuresdkdocs.blob.core.windows.net/$web` and need to be accessed via `https://azuresdkdocs.z19.web.core.windows.net`